apollo-normalized-cache-sqlite/src/commonMain/kotlin/com/apollographql/apollo/cache/normalized/sql/SqlNormalizedCache.kt

```diff
@@ -1,153 +1,88 @@
 package com.apollographql.apollo.cache.normalized.sql
 
-import com.apollographql.apollo.api.internal.json.BufferedSourceJsonReader
-import com.apollographql.apollo.api.internal.json.JsonReader
 import com.apollographql.apollo.cache.ApolloCacheHeaders.EVICT_AFTER_READ
 import com.apollographql.apollo.cache.CacheHeaders
 import com.apollographql.apollo.cache.normalized.CacheKey
 import com.apollographql.apollo.cache.normalized.NormalizedCache
 import com.apollographql.apollo.cache.normalized.Record
 import com.apollographql.apollo.cache.normalized.RecordFieldJsonAdapter
-import okio.Buffer
-import okio.IOException
+import com.apollographql.apollo.cache.normalized.sql.internal.RecordsetDataSource.deleteAllRecords
+import com.apollographql.apollo.cache.normalized.sql.internal.RecordsetDataSource.deleteRecord
+import com.apollographql.apollo.cache.normalized.sql.internal.RecordsetDataSource.selectAllRecords
+import com.apollographql.apollo.cache.normalized.sql.internal.RecordsetDataSource.selectRecord
+import com.apollographql.apollo.cache.normalized.sql.internal.RecordsetDataSource.selectRecords
+import com.apollographql.apollo.cache.normalized.sql.internal.RecordsetDataSource.updateRecords
 import kotlin.reflect.KClass
 
 class SqlNormalizedCache internal constructor(
-    private val recordFieldAdapter: RecordFieldJsonAdapter,
     private val database: ApolloDatabase,
-    private val cacheQueries: CacheQueries
```

Comment:
Should we keep the `cacheQueries` parameter instead of `database` ? Looks like we don't use anything from `database` downstream.